maestro-test/src/test/resources/051_set_location.yaml

```diff
@@ -0,0 +1,7 @@
+appId: com.example.app
+---
+- launchApp:
+    appId: com.example.app
+- setLocation:
```

Comment:
Totally optional, but it would be great to have an additonal alias version that just accepts 2 numbers separated by comma: `setLocation: 13.5266,78.2150`

Reply:
Tracking this feature on https://github.com/mobile-dev-inc/maestro/issues/254. We can pick this up after adding Android support.